fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -68686,6 +68680,304 @@ relationship: overlaps FBbt:00007053 ! lateral horn
+synonym: "larval serotonergic SE2-3 and SE2-4" NARROW [FlyBase:FBrf0219731]
+is_a: FBbt:00110950 ! larval serotonergic SE2 neuron
+created_by: CP
+creation_date: 2018-06-25T13:56:58Z
+
+[Term]
+id: FBbt:00047323
+name: larval serotonergic anterior ventromedial SE3 neuron
+def: "Larval serotonergic SE3 neuron whose cell body is located in the labial segment, in an anterior ventromedial position. There are usually three of these cells. They cells bifurcate close to the cell body, sending branches ipsilaterally and contralaterally. The ipsilateral branch remains restricted to the ventromedial part of the ipsilateral subesophageal zone. The contralateral branch splits more extensively  to innervate more of the contralateral subesophageal zone (Huser et al., 2012)." [FlyBase:FBrf0048551, FlyBase:FBrf0213697, FlyBase:FBrf0219731]
+synonym: "larval serotonergic SE3-1 and SE3-2" NARROW [FlyBase:FBrf0219731]
+is_a: FBbt:00110952 ! larval serotonergic SE3 neuron
+created_by: CP
+creation_date: 2018-06-25T14:33:38Z
+
+[Term]
+id: FBbt:00047324
+name: larval serotonergic posterolateral SE3 neuron
+def: "Larval serotonergic SE3 neuron whose cell body is located in the labial segment, in a more posterolateral position. A side branch of the primary neurite innervates the ventrolateral part of the subesophageal zone, while a second side branch arborizes in its dorsomedial portion before crossing the midline. The contralateral pattern consists of a dorsal process extending laterally to the middle of the neuromere before turning sharply anterior and branching." [FlyBase:FBrf0219731]
+synonym: "larval serotonergic SE3-3 and SE3-4" NARROW [FlyBase:FBrf0219731]
+is_a: FBbt:00110952 ! larval serotonergic SE3 neuron
+created_by: CP
+creation_date: 2018-06-25T14:36:34Z
+
+[Term]
+id: FBbt:00047325
+name: adult serotonergic AP neuron
+def: "Serotonergic neuron whose cell body is located in the anterior protocerebrum. There are two to three cells in this cluster." [FlyBase:FBrf0213697]
+is_a: FBbt:00005106 ! neuron
+relationship: capable_of GO:0060096 ! serotonin secretion, neurotransmission
+relationship: has_soma_location FBbt:00111151 ! cell body rind of anterior neuropils
+created_by: CP
+creation_date: 2018-06-25T15:24:42Z
+
+[Term]
+id: FBbt:00047326
+name: adult serotonergic DP neuron
+def: "Serotonergic neuron whose cell body is located in the dorsal protocerebrum. There is one of these cells per hemisphere." [FlyBase:FBrf0213697]
+is_a: FBbt:00005106 ! neuron
+relationship: capable_of GO:0060096 ! serotonin secretion, neurotransmission
+relationship: has_soma_location FBbt:00100308 ! cell body rind of superior neuropils
+relationship: part_of FBbt:00005095 ! brain
+created_by: CP
+creation_date: 2018-06-25T15:24:57Z
+
+[Term]
+id: FBbt:00047329
+name: serotonergic T1-3 neuron
```

Comment:
Worried this name may be a little confusing.  I initially read it as referring to segments T1-T3.  Perhaps rename to T1.3 and make T1-3 a synonym?

Reply:
Ditto - related classes/names.

Reply:
Now looks good. Your solution is better.